Brooks/GOOLFinallyTagless.cabal

```diff
@@ -0,0 +1,37 @@
+Name:		GOOLFinallyTagless
```

Comment:
Do we want to keep this as a separate package? [It needs a better name!]. It might indeed be a good idea to do so.

Maybe just GOOL.cabal?